src/sys/wait.rs

```diff
@@ -1,38 +1,83 @@
-use libc::{self, c_int};
+//! This module contains the `wait()` and `waitpid()` functions.
+//!
+//! These are used to wait on and obtain status information from child processes, which provide
+//! more granular control over child management than the primitives provided by Rust's standard
+//! library, and are critical in the creation of shells and managing jobs on *nix platforms.
+//!
+//! Manual Page: http://pubs.opengroup.org/onlinepubs/007908799/xsh/wait.html
+//!
+//! # Examples
```

Comment:
Please provide a brief description of what this code does, either in text before the example or using comments within the code.